Wrappers/Python/test/test_functions.py

```diff
@@ -970,7 +969,47 @@ def test_call_real_anisotropic(self):
         
         res1 = self.tv_aniso(x_real)
         res2 = self.grad.direct(x_real).pnorm(1).sum()
-        np.testing.assert_equal(res1, res2)                
+        np.testing.assert_equal(res1, res2)     
+
+    def test_strongly_convex_TV(self):
```

Comment:
You showed us a nice demo of using the strong convexity constant but in this PR I see only the unit test.  It is a bit difficult to review without a demo to run or at least look at. Where is that demo, could it be included or put somewhere?

Reply:
Demo is [here](https://github.com/epapoutsellis/PDHG_SPDHG/blob/master/Benchmark_TVTomoRecons.ipynb).

Reply:
Thanks - I think we said to create a new folder in top level of CIL-Demos with misc scripts and add the demos there. I can't remember the folder name we agreed on though, can you @gfardell?

Reply:
Add the demo to CIL-Demos, see https://github.com/TomographicImaging/CIL-Demos/issues/57

Reply:
Added in [misc](https://github.com/TomographicImaging/CIL-Demos/tree/main/misc).